Makefile

```diff
@@ -26,8 +26,10 @@ install:
 	  done
 	install -D -m 0644 -t $(DESTDIR)/$(prefix)/lib/systemd/system systemd/*.service systemd/*.timer systemd/*.path systemd/*.target
 	install -d -m 0755 $(DESTDIR)/$(prefix)/lib/systemd/system/multi-user.target.wants
+	install -d -m 0755 $(DESTDIR)/$(prefix)/lib/systemd/system/bootc-multi-user.service.wants
 	ln -s ../bootc-status-updated.path $(DESTDIR)/$(prefix)/lib/systemd/system/multi-user.target.wants/bootc-status-updated.path
-	ln -s ../bootc-status-updated.target $(DESTDIR)/$(prefix)/lib/systemd/system/multi-user.target.wants/bootc-status-updated.target
+	ln -s ../bootc-status-updated.target $(DESTDIR)/$(prefix)/lib/systemd/system/bootc-multi-user.service.wants/bootc-status-updated.target
```

Comment:
This can just be a `Wants=bootc-status-updated.target` directly in the `.service` file right?

Reply:
re: below, I'll just drop the `.service` entirely, and then we'll do the static enable of `bootc-status-updated-onboot.target` -> `multi-user.target`.